README.md

```diff
@@ -29,3 +32,70 @@ To test if it is installed correctly,
 ```julia
 pkg> test ArchGDAL
 ```
+
+## Getting Involved
+
+### Community
+
+This package will not be possible without [JuliaLang](https://julialang.org/), [GDAL](https://github.com/OSGeo/gdal) and [GDAL.jl](https://github.com/JuliaGeo/GDAL.jl). They are maintained by https://julialang.org/community/, https://www.osgeo.org/ and https://juliageo.org/ respectively. In case of any contention for support and involvement, we encourage participation and contributions to those projects and communities over this package.
+
+### Style Guide
+
+ArchGDAL.jl uses [JuliaFormatter.jl](https://github.com/domluna/JuliaFormatter.jl) as
+an autoformatting tool, and uses the options in [`.JuliaFormatter.toml`](https://github.com/yeesian/ArchGDAL.jl/blob/master/.JuliaFormatter.toml).
+
+If you wish to format code, `cd` to the ArchGDAL.jl directory, then run:
+```julia
+] add JuliaFormatter
+using JuliaFormatter
+format(".")
+```
+
+### Dependencies
+To manage the dependencies of this package, we work with [environments](https://pkgdocs.julialang.org/v1/environments/):
+
+1. Navigate to the directory corresponding to the package:
+
+```julia
+shell> cd /Users/yeesian/.julia/dev/ArchGDAL
+/Users/yeesian/.julia/dev/ArchGDAL
+```
+
+2. Activate the environment corresponding to `Project.toml`):
```

Comment:
Nitpick; This misses the step of actually starting Julia. Which you could do with `julia --project`

Reply:
Yeah that's a good point -- I did it in the REPL's shell mode, so I forgot documenting this step for others. Updated in https://github.com/yeesian/ArchGDAL.jl/pull/179/commits/26a8010f1e904bda8ce345792aac1d29297f5af5.